ls1intum/Artemis
# [Feature] Exercise dashboard button in exercise details pages

### Checklist

### Motivation and Context


In the exercise details pages I added a button to directly access the exercise-assessment dashboard.
Furthermore I cleaned up the button layout a bit. There is still some room for further improvements.

### Description


### Steps for Testing


1. Navigate to the course oveview -> exercises
2. Click on an exercise of every type (directly on the name), and check out the exercise detail page (Text, Modeling, Programming, File-Upload)
3. Test every button on those pages
4. Now, in an exam with a programming exercise, check the exercise details page too.

### Screenshots


File-Upload-exercise details page:
![image](https://user-images.githubusercontent.com/33342534/123426577-dee7bc00-d5c3-11eb-8b4f-7f9520bf3570.png)
Programming exercise details page:
![image](https://user-images.githubusercontent.com/33342534/123426651-f3c44f80-d5c3-11eb-9a95-5da08c68d4c5.png)
Modeling & text exercise details page: (they are identical)
![image](https://user-images.githubusercontent.com/33342534/123426707-076fb600-d5c4-11eb-8204-11cb107279a4.png)
